lib/kml/networklink.ts

```diff
@@ -0,0 +1,232 @@
+import type { Feature, Polygon } from "geojson";
+import type { KMLOptions } from "lib/kml";
+import { type StyleMap, get1, getMulti, num1 } from "../shared";
+import { extractIconHref, extractStyle } from "./extractStyle";
+import {
+    type Schema,
+    extractCascadedStyle,
+    extractExtendedData,
+    extractTimeSpan,
+    extractTimeStamp,
+    getMaybeHTMLDescription,
+} from "./shared";
+
+interface BoxGeometry {
+    bbox?: BBox;
+    geometry: Polygon;
+}
+
+enum AltitudeMode {
+    ABSOLUTE = "absolute",
+    RELATIVE_TO_GROUND = "relativeToGround",
+    CLAMP_TO_GROUND = "clampToGround",
+    CLAMP_TO_SEAFLOOR = "clampToSeaFloor",
+    RELATIVE_TO_SEAFLOOR = "relativeToSeaFloor",
+}
+
+type BBox = [number, number, number, number];
+
+type LOD = [number, number | null, number | null, number | null];
+interface IRegion {
+    coordinateBox: BoxGeometry | null;
+    lod: LOD | null;
+}
+
+function getNetworkLinkRegion(node: Element): IRegion | null {
+    const region = get1(node, "Region");
+
+    if (region) {
+        return {
+            coordinateBox: getLatLonAltBox(region),
+            lod: getLod(node),
+        };
+    }
+    return null;
+}
+
+function getLod(node: Element): LOD | null {
+    let resLOD: LOD | null = null;
+
+    const lod = get1(node, "Lod");
+
+    if (lod) {
+        resLOD = [
+            num1(lod, "minLodPixels") ?? -1,
+            num1(lod, "maxLodPixels") ?? -1,
+            num1(lod, "minFadeExtent") ?? null,
+            num1(lod, "maxFadeExtent") ?? null,
+        ];
+    }
+
+    return resLOD;
+}
```

Comment:
```suggestion
function getLod(node: Element): LOD | null {
    const lod = get1(node, "Lod");

    if (lod) {
        return [
            num1(lod, "minLodPixels") ?? -1,
            num1(lod, "maxLodPixels") ?? -1,
            num1(lod, "minFadeExtent") ?? null,
            num1(lod, "maxFadeExtent") ?? null,
        ];
    }

    return null;
}
```

I think this gets the same behavior with a tad less typing & complexity?